proxy/wireguard/tun_linux.go

```diff
@@ -199,6 +199,7 @@ func createKernelTun(localAddresses []netip.Addr, mtu int, handler promiscuousMo
 
 		r := netlink.NewRule()
 		r.Table, r.Family, r.Src = ipv6TableIndex, unix.AF_INET6, addr.IPNet
+		r.Table, r.Family, r.OifName = ipv6TableIndex, unix.AF_INET6, n
```

Comment:
```suggestion
		r.OifName = n
```

those other things are already assigned